manual.asc

```diff
@@ -937,18 +937,24 @@ To stop receiving notifications go to Deck options > Reminders and unmark the ch
 
 [[betaTesting]]
 === Beta testing
-If you want to try out the latest features in AnkiDroid, you can sign up for the beta testing program as follows:
+
+Beta releases :: If you want to try out the latest features in AnkiDroid, you can sign up for the beta testing program as follows:
 
  . Visit the https://play.google.com/apps/testing/com.ichi2.anki[Google Play Beta page]
  . Click *Become a beta tester*
  
-After following these steps, the latest beta version will automatically be installed by Google Play in the same way as ordinary updates.
-If you are more adventurous, you can also become an alpha tester, by joining the https://groups.google.com/forum/#!forum/ankidroidalphatesters[alpha testers group]
-in addition to performing the above steps for beta testing.
+Alpha releases :: If you are more adventurous, you can also become an alpha tester, by joining the https://groups.google.com/forum/#!forum/ankidroidalphatesters[alpha testers group]
+in addition to performing the above steps for beta testing. The group itself receives no messages, it is used exclusively to determine which Google accounts Google Play will deliver alpha releases to.
+
+Beta / alpha / stable release compatibility :: The 2.8.x releases and the 2.9.x releases are forwards and backwards compatible, with no known risk of data loss. It should be safe to try a 2.9.x alpha or beta while preserving the ability to leave the program and use 2.8.x on the same collection.
+
+Release delivery :: After following these steps, Google Play will automatically deliver the most recent available beta (or alpha) release to your device in the same way as ordinary updates, after some delay. +
++
+To get a beta or alpha update with no delay instead of waiting, may manually tap the button to scan for updates in the Google Play Store app on your device, then update via the update button in the list of app updates found (not in the app details section).
```

Comment:
Again, too specific IMO. This isn't a manual for how to use the current version of Google Play.

Reply:
You can link to this:

https://support.google.com/googleplay/answer/113412?hl=en